mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -388,6 +383,8 @@ def tuneMLIRKernels(configs, xdlops):
     if len(sys.argv) == 1:
         # batch benchmark with MLIR and MIOpen.
         generatePerformanceResults(configs, xdlops)
+    elif sys.argv[1] == '-bmlir_only':
+        generatePerformanceResults(configs, xdlops, skipMIOpen=True)
```

Comment:
This is the same as the option "-b" ?

Reply:
No, because it'll generate performance reports acceptable to the report generator? 

Although I'll go and test if the output of -b goes through the perf report maker without problems